hypothesis-python/tests/pandas/test_series.py

```diff
@@ -61,3 +63,23 @@ def test_unique_series_are_unique(s):
 @given(pdst.series(dtype="int8", name=st.just("test_name")))
 def test_name_passed_on(s):
     assert s.name == "test_name"
+
+
+def test_pandas_nullable_types():
+    st = pdst.series(dtype=pandas.core.arrays.integer.Int8Dtype())
+    e = find_any(st, lambda s: s.isna().any())
+    assert type(e.dtype) == pandas.core.arrays.integer.Int8Dtype
+
+
+def test_pandas_nullable_types_class():
+    with pytest.raises(
+        InvalidArgument, match="Otherwise it would be treated as dtype=object"
+    ):
+        st = pdst.series(dtype=pandas.core.arrays.integer.Int8Dtype)
+        find_any(st, lambda s: s.isna().any())
```

Comment:
This test could instead be a new case in one of the big parameter lists in `test_argument_validation.py` (a sibling file).

Reply:
Done